README.md

```diff
@@ -312,6 +312,18 @@ Period Range
  .InRelativePeriodRange(Period.FromWeeks(2), Period.FromDays(20))
 ```
 
+### UnitOfMeasure and AggregationRule extraction for Actual and Versioned Time Series
```

Comment:
these are 2 separate features.

AggregationRule override on query and UoM convertion. Separate them.